src/globals.cpp

```diff
@@ -50,7 +50,7 @@ int BetterSMS::getScreenRenderWidth() {
     case AspectRatioSetting::WIDE:
         return 700;
     case AspectRatioSetting::ULTRAWIDE:
-        return 1050;
+        return 833;
```

Comment:
Any particular reason for this value?

Reply:
How orthographic coordinates should be calculated, in ways to scale at screen center, is to use the formula like this:

(0.5*(r + l)*scale) + (0.5*(r + l) - l);

Take 600 for example, if you scale it to 1:1, multiplying that value by 0.75, would give -75 for left, and 525 for right.

Reply:
That's fine and all, but if you want to return 833 here, we need to adjust widescreen.cpp to account for the change. Right now this breaks the ultra-wide scale such that everything looks slightly more wide than widescreen. And as such, WIDE should also have its value adjusted no?